domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Behandlingsinformasjon.kt

```diff
@@ -243,6 +249,17 @@ data class Behandlingsinformasjon(
         override fun isComplete(): Boolean = true
     }
 
+    data class Ektefelle(val harEktefellePartnerSamboer: Boolean, val fnr: Fnr?) : Base() {
```

Comment:
Dette var rart, hvorfor kan vi ikke bare ha data class Ektefelle(val fnr: Fnr)

Reply:
Vi har jo allerede en EktefellePartnerEllerSamboer typer. Hva gjør at de er forskjellige konsepter?

Reply:
> Dette var rart, hvorfor kan vi ikke bare ha data class Ektefelle(val fnr: Fnr)

Som `Behandlingsinformasjon` er byggd nå så indikerar `null` att saksbehandlaren ikke har svarat på spørsmålet. Därför må vi ha ett fält som sier om saksbehandlaren har svarat Ja / Nej (`harEktefellePartnerSamboer`), och sen isåfall vilket fnr den ektefellen har.

> Vi har jo allerede en EktefellePartnerEllerSamboer typer. Hva gjør at de er forskjellige konsepter?
Ja, idéellt sätt så hade man helst brukt samma type. Vet ikke helt hvordan man hade gjort det här eftersom forskjellen på de två är att `EktefellePartnerEllerSamboer` är en type før hvordan *søkaren* vill specificera sin ektefelle. Medans `Ektefelle` här i `Behandlingsinformajson.kt` är typen før hvordan *saksbehandlaren* vurderer kriteriet/vilkåret. Om det gav mening? 😅

Reply:
Kunne vi delt de inn i en sealed class i domenet i det minste? Kan godt beholde null konseptet for ikke svart, men at det enten er EktefellePartnerEllerSamboer helst (vi burde vel kreve de samme verdiene?) eller IngenEktefelle

Reply:
Ja, skal se på det. Liker också att ha det sån egentligen.